Tools/autotest/rover.py

```diff
@@ -5352,11 +5367,13 @@ def test_scripting_auxfunc(self):
         self.context_collect("STATUSTEXT")
         self.set_parameters({
             "SCR_ENABLE": 1,
-            "RELAY_PIN": 1,
+            "RELAY1_FUNCTION": 1
         })
         self.install_example_script_context("RCIN_test.lua")
         self.reboot_sitl()
 
+        self.set_parameter("RELAY1_PIN", 1)
```

Comment:
A comment as to why this isn't done in the `set_parameters` call above might be appropriate?

Reply:
Now works in `set_parameters`